### PR TITLE
Added forward bias voltage

### PR DESCRIPTION
**Pull request by sadiqebrahim**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: electronics/forward_bias_current.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from math import e
+
```
Comment (CaedenPH) [inline electronics/forward_bias_current.py:2]:
```suggestion
```

**Review by CaedenPH — APPROVED** [review]
Looks good

**Comment by CaedenPH** [comment]
@cclauss I know this is on hold but its a pretty quick and easy one to merge

File: electronics/forward_bias_current.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+from scipy.constants import Boltzmann, physical_constants
+
+T = 300  # TEMPERATURE (unit = K)
+
+
+def forward_bias_current(
+    voltage: float,  # Forward voltage
+    rev_sat_current: float,  # Reverse Saturation Current
```
Comment (cclauss) [inline electronics/forward_bias_current.py:10]:
```suggestion
    forward_voltage: float,
    reverse_saturation_current: float,
```

File: electronics/forward_bias_current.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+from math import e
+
+from scipy.constants import Boltzmann, physical_constants
```
Comment (tianyizheng02) [inline electronics/forward_bias_current.py:3]:
```suggestion
from scipy import constants
```
`constants` already has an easy way to get the value of constants without needing to access the underlying `physical_constants` dict: `constants.get()`

File: electronics/forward_bias_current.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        raise ValueError("Reverse saturation current cannot be greater than 1e-5")
+    else:
+        return reverse_saturation_current * (
+            e
+            ** (
+                (forward_voltage * physical_constants["electron volt"][0])
+                / (Boltzmann * T)
+            )
+            - 1
+        )
```
Comment (tianyizheng02) [inline electronics/forward_bias_current.py:46]:
```suggestion
    if forward_voltage < 0:
        raise ValueError("Forward voltage cannot be negative")
    if reverse_saturation_current <= 0:
        raise ValueError("Reverse saturation current must be positive")
    if reverse_saturation_current > 1e-5:
        raise ValueError("Reverse saturation current cannot be greater than 1e-5")
    return reverse_saturation_current * (
        exp(
            forward_voltage * constants.get("electron volt") / (constants.k * T)
        ) - 1
    )
```

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]

File: electronics/forward_bias_current.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+T = 300  # TEMPERATURE (unit = K)
+
+
+def forward_bias_current(
+    forward_voltage: float,
+    reverse_saturation_current: float,
+) -> float:
+    """
+    This function can calculate the Forward Bias Current of a pn junction diode.
+    This is calculated from the given two values.
```
Comment (cclauss) [inline electronics/forward_bias_current.py:14]:
```suggestion
    Calculate the Forward Bias Current of a pn junction diode from given two values.
```

**Pull request closed without merging**